oop/src/chapter10/Solution107.java

```diff
@@ -0,0 +1,54 @@
+package chapter10;
+
+import chapter9.Solution97;
+
+import java.util.Scanner;
+
+public class Solution107 {
+    public static void main(String[] args) {

```

Comment:
Nagyon szórakoztató kis program, mondhatni, ez is egy játék, mint amiket már írtál korábban 😊